app.R

```diff
@@ -404,12 +433,30 @@ server <- function(input, output) {
            fill = "Target Audience",
            title = "Visitor Distribution across Educational Resources") +
       scale_fill_manual(values=cbPalette)
-   })
+  })
   # Caption
   output$unique_visitor_website_caption <- renderText({
     "Number of visitors for various educational resources, segmented by different target audience categories: everyone, leadership, new to data, and software developers."
   })
   
+  # Course Engagement By Modality
+  output$engagement_by_modality <- renderPlot({
+    course_processed()  %>% 
+      ggplot(aes(x = forcats::fct_reorder(course_name, course_order),
+                 y = number_of_learners, fill = `Target Audience`,)) +
+      geom_col() + 
+      facet_wrap(~modality, nrow = 3, scales = "free_y")  + 
+      scale_fill_manual(values=cbPalette) + 
+      theme_classic() + 
+      theme(axis.text.x = element_text(angle = 45, hjust=1),
+            text = element_text(size = 17, family = "Arial")) + 
+      labs(x = NULL, y = "Number of Learners")
+  })
+  # Caption
+  output$engagement_by_modality_caption <- renderText({
+    "The number of learners for each modality (website = bookdown website) for each course."
```

Comment:
```suggestion
    "The number of learners for each modality for each course."
```

I think most folks don't know what we mean by bookdown and not sure it adds clarity. I could be wrong though so happy to figure out a better way to add this note.

Reply:
I deleted the `website = bookdown website`. I admit, it might get confusing.